README.md

```diff
@@ -136,12 +161,12 @@ Returns the connected signer.
 const signer = safeSdk.getSigner()
 ```
 
-### getAddress
+### getSafeAddress
 
 Returns the address of the current Safe Proxy contract.
 
 ```js
-const address = safeSdk.getAddress()
+const address = safeSdk.getSafeAddress()
```

Comment:
This feels a little weird as the `safeSdk` is an `EthersSafe` (so in the future maybe even only a `Safe` when we have web3 and ethers support) and calling `getSafeAddress` on that feels like a duplicate. I would leave it at `getAddress`, and just be explicit for the multiSend address